### PR TITLE
[BOOST-577] End-to-End tests - change cart mutation

### PR DESCRIPTION
**Pull request by tainguyenbui**
## Description
Create end-to-end test to check that the `ChangeCartItem` mutation works as expected.

## Changes
- Created tests that perform a `ChangeCartItem` mutation and read changes from its respective read model
- Refactored CLI integration test

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [ ] Updated documentation accordingly at [the docs repo](https://github.com/boostercloud/docs)
- [ ] Added a link pointing to the documentation PR in this PR

### CODE REVIEW

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by alvaroloes** [comment]
bot: integration

**Comment by tainguyenbui** [comment]
bot: integration

File: packages/framework-integration-tests/integration/end-to-end/.mocharc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff: true
+require: 'ts-node/register'
+extension:
+  - ts
+package: './package.json'
+recursive: true
+reporter: 'spec'
+timeout: 600000
```
Comment (alvaroloes) [inline packages/framework-integration-tests/integration/end-to-end/.mocharc.yml:8]: I would double this time. Integration tests now are almost reaching this time

File: packages/framework-integration-tests/integration/end-to-end/cart.integration.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            ChangeCartItem(input: { cartId: $cartId, productId: $productId, quantity: $quantity })
+          }
+        `,
+      })
+
+      expect(response).not.to.be.null
+      expect(response?.data?.ChangeCartItem).to.be.true
+    })
+
+    describe('Read models', () => {
```
Comment (alvaroloes) [inline packages/framework-integration-tests/integration/end-to-end/cart.integration.ts:35]: Was this `describe('Read models')` intended to be placed inside the `describe('Commands')`? It seems there is no relation between them

File: packages/framework-integration-tests/integration/end-to-end/cart.integration.ts [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+      context('1 cart item', () => {
+        const mockCartId: string = random.uuid()
+        const mockProductId: string = random.uuid()
+        const mockQuantity: number = random.number({ min: 1 })
+
+        beforeEach(async () => {
```
Comment (alvaroloes) [inline packages/framework-integration-tests/integration/end-to-end/cart.integration.ts:41]: Wow! Clever way to provision data for each test 👏

File: packages/framework-integration-tests/integration/end-to-end/cart.integration.ts (hunk 2 above)
Comment (tainguyenbui) [reply packages/framework-integration-tests/integration/end-to-end/cart.integration.ts:35]: it was intended to reflect that a Command makes changes in a read model. In my head, I was thinking that a Command in the future might have an impact over another thing that not necessarily has to be a read model.

**Pull request merged**